<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createkeyfile</class>
 <widget class="QWidget" name="createkeyfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEditFileName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>key file name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>path to a folder to create a key in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOpenFolder">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>open a folder a key file will be created in</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCreate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>destination folder</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
